webapp/components/Comment.vue

```diff
@@ -28,13 +27,27 @@
       </no-ssr>
       <!-- eslint-disable vue/no-v-html -->
       <!-- TODO: replace editor content with tiptap render view -->
+
+      <div v-if="isCollapsed" v-html="comment.contentExcerpt" style="padding-left: 40px;" />
+      <div
+        v-show="comment.content !== comment.contentExcerpt"
+        style="text-align: right;  margin-right: 20px; margin-top: -12px;"
+      >
+        <a v-if="isCollapsed" style="padding-left: 40px;" @click="isCollapsed = !isCollapsed">
+          {{ $t('comment.show.more') }}
+        </a>
+      </div>
+      <div v-if="!isCollapsed" v-html="comment.content" style="padding-left: 40px;" />
+      <div style="text-align: right;  margin-right: 20px; margin-top: -12px;">
+        <a v-if="!isCollapsed" @click="isCollapsed = !isCollapsed" style="padding-left: 40px; ">
+          {{ $t('comment.show.less') }}
+        </a>
+      </div>
       <ds-space margin-bottom="small" />
-      <div style="padding-left: 40px;" v-html="comment.contentExcerpt" />
-      <!-- eslint-enable vue/no-v-html -->
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-03T05:59:54Z" title="Wednesday, July 3rd 2019, 7:59:54 am +02:00">Jul 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -22,11 +22,33 @@
     </no-ssr>
     <!-- eslint-disable vue/no-v-html -->
     <!-- TODO: replace editor content with tiptap render view -->
-    <ds-space margin-bottom="small" />
-    <div style="padding-left: 40px;" v-html="comment.contentExcerpt" />
-    <!-- eslint-enable vue/no-v-html -->
+
+    <div v-if="isCollapsed" v-html="comment.contentExcerpt" style="padding-left: 40px;" />
+    <div
+      v-show="comment.content !== comment.contentExcerpt"
+      style="text-align: right;  margin-right: 20px; margin-top: -12px;"
+    >
+      <span
+        v-if="isCollapsed"
+        style="padding-left: 40px; color: #17b53f; cursor:pointer"
+        @click="isCollapsed = !isCollapsed"
+      >
+        {{ $t('comment.show.more') }}
+      </span>
+    </div>
+    <div v-if="!isCollapsed" v-html="comment.content" style="padding-left: 40px;" />
+    <div style="text-align: right;  margin-right: 20px; margin-top: -12px;">
+      <span
+        v-if="!isCollapsed"
+        @click="isCollapsed = !isCollapsed"
+        style="padding-left: 40px; color: #17b53f; cursor:pointer"
+      >
+        {{ $t('comment.show.less') }}
+      </span>
+    </div>
```


Like @roschaefer said. Would be nice to make a component out this.
But my opinion is to make an issue for the future, after we replaced the alpha. @ogerly @roschaefer 

PS: And we have to use the editor functionality to display this content in the future, if I have understood @appinteractive right. Therefor this has to be consider if we do make an own component.